### DISCUSSION TITLE
Mocap: Optitrack

### DISCUSSION THREAD

**Discussion by sesem738** [bitcraze/discussions#948] [Q&A]
I am trying to use an Optitrack system to obtain pose information. I followed the mocap_hl_commander example, but I encountered some issues. When I set the mocap_system_type to vrpn, I can receive data from the Optitrack. However, when I change the mocap_system_type to optitrack, nothing happens. I am not sure if this is a problem with my configuration or something else. Can anyone help me with this?

**Comment by knmcguire** [comment]
We have had several posts about this recently. Perhaps it is good to first check out the solutions that other people tried out: https://github.com/orgs/bitcraze/discussions?discussions_q=is%3Aopen+label%3A%22Positioning%3A+MoCap%22

Let us know if any of those doesn't solve your issue.
